internal/postgres/config.go

```diff
@@ -246,6 +247,18 @@ func startupCommand(
 		}
 	}
 
+	pg_rewind_override := ""
+	if config.FetchKeyCommand(&cluster.Spec) != "" {
+		// Quoting "EOF" disables parameter substitution during write.
+		// - https://tldp.org/LDP/abs/html/here-docs.html#EX71C
+		pg_rewind_override = `cat << "EOF" > /tmp/pg_rewind_tde.sh
+#!/bin/sh
+pg_rewind -K "$(postgres -C encryption_key_command)" "$@"
```

Comment:
Does the `"$@"` get used?

Reply:
Yes, that allows the other required flags to be passed to `pg_rewind` when it's invoked by Patroni.

Reply:
That makes sense. Thanks!